app/Http/Middleware/VerifyCsrfToken.php

```diff
@@ -14,4 +15,13 @@ class VerifyCsrfToken extends Middleware
     protected $except = [
         'sms/receive',
     ];
+
+    public function handle($request, Closure $next)
+    {
+        if (env('APP_ENV') === 'testing') {
+            return $next($request);
+        }
```

Comment:
I think this already gets disabled somewhere at the framework level but I could be wrong

Reply:
Yeah this was an attempt to get some tests running locally, they're all returning 419's for some reason, but it's not happening in CI so /shrug